File: Berdan_Akyurek.rs
use rand::Rng;
fn main(){
    let number = 30;
    let mut tries = 0;
    //let mut rng = rand::thread_rng();
    let mut assumption = -1;
    while assumption != number {
        assumption = rand::thread_rng().gen_range(0, 100);
        tries = tries + 1;
        println!("{}",assumption);
    }
    println!("{} {}",tries, "tries");
}

File: HW2/Berdan_Akyurek.dart
import 'dart:math';

void main() {
  int number = 30;
  int tries = 0;
  var rnd = new Random();
  var assumption;
  do {
    assumption = rnd.nextInt(100);
    print(assumption);
    tries++;
  } while (assumption != number);
  print("$tries tries.");
}
